src/ui/actions/session.ts

```diff
@@ -270,8 +269,6 @@ function onLoadingFinished(): UIThunkAction {
       await ThreadFront.waitForSession();
       await ThreadFront.initializedWaiter.promise;
       await ThreadFront.ensureAllSources();
-
-      ThreadFront.ensureCurrentPause();
```

Comment:
AFAICT, this is not needed, since we `warp` in `jumpToIniitalPausePoint`? Also, it was getting called before we were done wrangling the focus window, which was causing more `failed to create pause, region has been unloaded` errors.